pio-engine/build.sbt

```diff
@@ -0,0 +1,11 @@
+import AssemblyKeys._
+
+assemblySettings
+
+name := "predictionio-load-forecasting"
+
+libraryDependencies ++= Seq(
+  "org.apache.predictionio"    %% "apache-predictionio-core" % "0.10.0-incubating" % "provided",
+  "org.apache.spark"  %% "spark-core"   % "1.3.0"           % "provided",
+  "org.apache.spark"  %% "spark-mllib"  % "1.3.0"           % "provided"
+)
```

Comment:
Also, I got `Failed to load implementation from com.github.fommil.netlib.nativesystemblas` in training. I checked the [netlib-java](https://github.com/fommil/netlib-java) GitHub page and added the dependency but has no luck.

I checked more on Github and it seems we also need to specify it in Sbt in run/compile time with something like `sbt -J"-Dcom.github.fommil.netlib.BLAS=com.github.fommil.netlib.NativeRefBLAS"  assembly
`